README.md

```diff
@@ -1,90 +1,130 @@
 # Flashbots
 
-Flashbots is a research and development organization formed to mitigate the negative externalities and existential risks posed by miner-extractable value (MEV) to smart-contract blockchains, starting with Ethereum. Our primary focus is to enable a *permissionless, transparent, and fair ecosystem* for MEV extraction. Flashbots arose out of the *MEV Pi-rate Ship*, a neutral, chain-agnostic, interdisciplinary research collective that supports MEV-related theoretical and empirical research. 
+Flashbots is a research and development organization working on mitigating the negative externalities of current MEV extraction techniques and avoiding the existential risks MEV could cause to state-rich blockchains like Ethereum. Our work falls under three efforts: Democratizing Access to MEV Revenue, Bringing Transparency to MEV Activity and Distributing MEV Revenue.
 
```

Comment:
NIT, but I see these as _goals_ rather than "efforts". I personally liked this from the previous version: "Our primary focus is to enable a *permissionless, transparent, and fair ecosystem* for MEV extraction."

Reply:
also, is one of the goals really to "distribute MEV revenue"? this can be a bit confusing.. at least I'd have "redistributing" :)

Reply:
### thank you! changes made :)